assist_microphone/rootfs/etc/s6-overlay/s6-rc.d/assist_microphone/finish

```diff
@@ -0,0 +1,26 @@
+#!/command/with-contenv bashio
+# shellcheck shell=bash
+# ==============================================================================
+# Take down the S6 supervision tree when service fails
+# s6-overlay docs: https://github.com/just-containers/s6-overlay
+# ==============================================================================
+declare exit_code
```

Comment:
Seems to be unused?

Reply:
Removed. It's there in other `finish` scripts, so I've made a task to clean those up too.